src/pages/index.js

```diff
@@ -112,11 +173,28 @@ const Home = () => {
           <p>
             Fusion.js gives you the developer experience you expect from a
             React/Redux setup and provides tools to take project quality to the
-            next level.<br />
-            <Link to="/docs/getting-started">
-              Let's get started &gt;
-            </Link>
+            next level.
           </p>
+          <CTAContainer>
+            <CTALeftBox>
+              <CTATitle>Try it out!</CTATitle>
+              <p>Get started building applications with Fusion.js in minutes.</p>
+              <CTAPrimaryButton to="/docs/getting-started" onClick={trackQuickStart}>Quick Start</CTAPrimaryButton>
+              <CTASecondaryButton href="https://github.com/fusionjs" target="_blank">Github</CTASecondaryButton>
```

Comment:
I personally don't really think that our org landing page is a very good place to direct users. I think this might change once we get the monorepo implemented, but feel like we might be better off leaving out the Github link for now. Up to you.

Reply:
Maybe it should go to https://github.com/fusionjs/create-fusion-app or something? We could improve the docs on that...

Reply:
Oh, this was actually something that Peter requested. Though he didn't have an opinion about which page specifically to go to. I'll check with him.

Reply:
Talked to Peter about it. We'll switch to fusion-core for now until we have a better place to link to but he wants to keep the Github link.